autotx/AutoTx.py

```diff
@@ -16,7 +18,13 @@
 @dataclass(kw_only=True)
 class Config:
     verbose: bool
-    logs_dir: Optional[str]
+    logs_dir: Optional[str] = None
+    log_costs: Optional[bool] = None
+    max_rounds: Optional[int] = None
+
+    def __post_init__(self) -> None:
```

Comment:
i dont think `__post_init__` is necessary here we could just to `max_rounds: int = 100` and if we want a different value we could just pass it initialization of `Config`

Reply:
Then we can't pass in a none value for it in Config (like we do in cli.py)

Reply:
seems weird to accept a `None` value when it [must be provided](https://github.com/polywrap/AutoTx/blob/a13557b71c1ca4b4ca2aa6672e387ed43d7b3926/autotx/AutoTx.py#L74-L75) - but I understand that this way we use the argument from the CLI, this is a nit so feel free to ignore :smile:

Reply:
It must be provided in AutoTx not in Config.
I've updated the Config class now, hopefully it's a bit clearer